src/views/product.js

```diff
@@ -16,24 +14,13 @@ export default class ProductView extends View {
   }
 
   /**
-   * check size of image until it is resolved, then set height of iframe.
+   * add event listener which triggers resize when the product image is loaded.
    */
-  resizeUntilLoaded() {
-    if (!this.iframe || !this.component.model.selectedVariantImage) {
-      return;
-    }
-    const img = this.wrapper.getElementsByClassName(this.component.classes.product.img)[0];
-    let intervals = 0;
-    if (img) {
-      const productResize = setInterval(() => {
```

Comment:
tbh I can't remember. I _think_ the offending browser was firefox, but im not certain. And I think it was that it was firing too soon. Or it might have had something to do with cached images not firing 'load' in one of the browsers? Yeah that might have been it. So if you have an image twice on one page (which does happen) it won't fire `load` the second time.